### PR TITLE
Publish from GitHub Actions

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Publish
+on:
+  push:
+    tags:
+      - "v0.*"
```
Comment (hiroyuki-sato) [inline .github/workflows/publish.yml:5]: nit: It looks like you forgot to update after `1.0` 😄 (Just comment)

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍

File: .github/workflows/publish.yml (hunk 1 above)
Comment (dmikurube) [reply .github/workflows/publish.yml:5]: Yeah, I'll take care of it! 😃

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**